db/schema.sql

```diff
@@ -1,13 +1,11 @@
-SELECT count(*) FROM information_schema.tables WHERE table_schema = CURRENT_SCHEMA() AND table_name = 'migrations' AND table_type = 'BASE TABLE';
-
 CREATE TABLE "migrations" (
     "id" varchar(255),
     PRIMARY KEY ("id")
 );
 
 CREATE TABLE "tenants" (
-    "created_at" timestamptz NOT NULL,
-    "updated_at" timestamptz,
+    "created_at" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" timestamptz NOT NULL,
     "deleted_at" timestamptz,
     "tenant_id" serial,
```

Comment:
should tenant id be a number? it kind of gives away how many tenants there are if they were to look at the header right?

Reply:
I don't think it's a concern. but if it is we could always start the autoincrement at a higher number :)